semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/TextDocumentOps.scala

```diff
@@ -635,6 +642,18 @@ trait TextDocumentOps { self: SemanticdbOps =>
               case gtree: g.TypeTreeWithDeferredRefCheck =>
                 traverse(gtree.check())
 
+              case gtree: g.Literal
+                  if isClassOf(gtree.pos.toMeta) && gtree.tpe != null && gtree.tpe.typeSymbol == g.definitions.ClassClass && gtree.pos.isRange =>
```

Comment:
nit: multiple lines make it easier to read the multiple conditions
```suggestion
                  if isClassOf(gtree.pos.toMeta) &&
                    gtree.tpe != null &&
                    gtree.tpe.typeSymbol == g.definitions.ClassClass &&
                    gtree.pos.isRange =>
```

Reply:
Fixed.